res/values/strings.xml

```diff
@@ -462,16 +462,16 @@
     <string name="pref_watch_mvbox_folder">Watch DeltaChat folder</string>
     <string name="pref_send_copy_to_self">Send copy to self</string>
     <string name="pref_auto_folder_moves">Automatic moves to DeltaChat folder</string>
-    <string name="pref_auto_folder_moves_explain">Chat conversations are moved to avoid cluttering the Inbox folder</string>
+    <string name="pref_auto_folder_moves_explain">Chat conversations are moved to avoid cluttering the \"Inbox\" folder</string>
     <string name="pref_email_interaction_title">Email interaction</string>
     <string name="pref_show_emails">Show classic emails</string>
     <string name="pref_show_emails_no">No, chats only</string>
     <string name="pref_show_emails_accepted_contacts">For accepted contacts</string>
     <string name="pref_show_emails_all">All</string>
     <string name="pref_empty_server_title">Delete e-mails from server</string>
-    <string name="pref_empty_server_msg">This function helps to free space on your IMAP server by deleting all e-mails, including chat-messages, in the given folders. Messages on this device will not be deleted. The deletion cannot be undone!</string>
-    <string name="pref_empty_server_inbox">Delete all e-mails in Inbox folder</string>
-    <string name="pref_empty_server_mvbox">Delete all e-mails in DeltaChat folder</string>
+    <string name="pref_empty_server_msg">This function helps free up space on your IMAP server by deleting all e-mails, including chat messages, in the given folders. Messages on this device will not be deleted. The deletion cannot be undone!</string>
+    <string name="pref_empty_server_inbox">Delete all e-mails in \"Inbox\" folder</string>
+    <string name="pref_empty_server_mvbox">Delete all e-mails in \"DeltaChat\" folder</string>
```

Comment:
Not sure "folder" adds anything here.

Reply:
I think it makes sense to leave it here, because the user has to distinguish between the app "Delta Chat" and the respective folder on the server.

Reply:
And this is the app?